zuuler/descarger.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Red Hat, Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+from argparse import ArgumentParser
+from argparse import Namespace
+from functools import partial
+import json
+from multiprocessing import cpu_count
+from multiprocessing import Pool
+import os.path
+from pathlib import Path
+from sys import exit
+
+import requests
+
+
+GITHUB_API_URL = 'https://api.github.com/'
+GITHUB_REPO_ENDPOINT = 'repos/{project}/'
+GITHUB_CONTENT_ENDPOINT = 'contents/{path}?ref={gitref}'
+
+OPENDEV_API_URL = 'https://opendev.org/api/v1/'
+OPENDEV_REPO_ENDPOINT = 'repos/{project}/'
+OPENDEV_CONTENT_ENDPOINT = 'contents/{path}?ref={gitref}'
+
+
+def get_raw_url_files_in_repository(repository: str,
+                                    data_required: dict,
+                                    branch: str = 'master',
+                                    errors_fatal: bool = True) -> dict:
+    if 'opendev.org' in repository:
+        ENDPOINT = (
+            OPENDEV_API_URL + OPENDEV_REPO_ENDPOINT + OPENDEV_CONTENT_ENDPOINT
+        )
+    elif 'github.com' in repository:
+        ENDPOINT = (
+            GITHUB_API_URL + GITHUB_REPO_ENDPOINT + GITHUB_CONTENT_ENDPOINT
+        )
+    else:
+        print('Unsupported repository')
```

Comment:
'Unsupported host. Supports: opendev.org and github.com repositories' gives me better idea what is the problem

Reply:
Makes sense, I updated the message, I will send the commit soon, thanks!